dbt/seeds/etl_fast_row_counts.csv

```diff
@@ -686,36 +686,36 @@ core_pudl__codes_datasources,,11
 core_pudl__codes_subdivisions,,69
 core_pudl__entity_plants_pudl,,19840
 core_pudl__entity_utilities_pudl,,16730
-core_sec10k__quarterly_company_information,1994,64352
-core_sec10k__quarterly_company_information,1995,69251
-core_sec10k__quarterly_company_information,1996,179590
-core_sec10k__quarterly_company_information,1997,410603
-core_sec10k__quarterly_company_information,1998,481589
-core_sec10k__quarterly_company_information,1999,388099
-core_sec10k__quarterly_company_information,2000,308338
-core_sec10k__quarterly_company_information,2001,282794
-core_sec10k__quarterly_company_information,2002,302341
-core_sec10k__quarterly_company_information,2003,269668
-core_sec10k__quarterly_company_information,2004,377608
-core_sec10k__quarterly_company_information,2005,378987
-core_sec10k__quarterly_company_information,2006,311654
-core_sec10k__quarterly_company_information,2007,288297
-core_sec10k__quarterly_company_information,2008,273039
-core_sec10k__quarterly_company_information,2009,382182
-core_sec10k__quarterly_company_information,2010,413124
-core_sec10k__quarterly_company_information,2011,327673
-core_sec10k__quarterly_company_information,2012,237836
-core_sec10k__quarterly_company_information,2013,229417
-core_sec10k__quarterly_company_information,2014,229801
-core_sec10k__quarterly_company_information,2015,219965
-core_sec10k__quarterly_company_information,2016,210391
-core_sec10k__quarterly_company_information,2017,191434
-core_sec10k__quarterly_company_information,2018,191965
-core_sec10k__quarterly_company_information,2019,186879
-core_sec10k__quarterly_company_information,2020,182356
-core_sec10k__quarterly_company_information,2021,195784
-core_sec10k__quarterly_company_information,2022,202021
-core_sec10k__quarterly_company_information,2023,193870
+core_sec10k__quarterly_company_information,1994,2286
```

Comment:
The relative row counts of years seems to be consistent with the previous (much larger) version of this table. It makes sense to me that this table is now much smaller.